phabricatoremails/render/templates/text/_macros.text.jinja2

```diff
@@ -0,0 +1,88 @@
+{% macro revision_info(revision) -%}
+{{ revision.link }}
+
+{% if revision.bug -%}
+(Associated with bug {{ revision.bug.id }}: {{ revision.bug.name }})
+({{ revision.bug.link }})
+{%- endif %}
+{%- endmacro %}
+
+{% macro secure_revision_info(revision) -%}
+{{ revision.link }}
+
+{% if revision.bug -%}
+(Associated with secure bug)
+({{ revision.bug.link }})
+{%- endif %}
+{%- endmacro %}
+
+{% macro reviewer_action(reviewer, revision) %}
+{%- if reviewer is soft_needed %}
+[!] A review is needed
+{%- elif reviewer is needed %}
+[!] Your review is needed
+{%- endif -%}
+{% endmacro %}
+
+{% macro footer() -%}
+-------------------------------
```

Comment:
Can you use "-"  * <number> here, so that we don't have to count or guess how many dashes are here.

If there's a common width that we can use here, maybe we should put this in a variable somewhere and use the same value everywhere. This can also reduce the number of arbitrary values we're using. Something like `max_width / 2` or `max_width - 10` etc... (see other conversations related to hard coding numbers and arbitrary numbers, also.)

Reply:
This footer line is defined once in a visual manner. Why do we need to count how many dashes that are here?
If we need to replicate this line in the future, we can move it into a macro accordingly.

Reply:
You're using some concept of "max width" already (e.g. the below "Email preferences" section). What my ask is here is to make this more consistent and obvious. You should probably use `textwrap` below also.